core/src/main/kotlin/org/evomaster/core/problem/rest/service/AIResponseClassifier.kt

```diff
@@ -56,8 +57,8 @@ class AIResponseClassifier : AIModel {
             else -> object : AIModel {
                 override fun updateModel(input: RestCallAction, output: RestCallResult) {}
                 override fun classify(input: RestCallAction) = AIResponseClassification()
-                override fun estimateAccuracy(endpoint: Endpoint): Double  = 0.0
-                override fun estimateOverallAccuracy(): Double = 0.0
+                override fun estimateMetrics(endpoint: Endpoint): ModelEvaluation  = ModelEvaluation(accuracy = 0.5,precision400 = 0.5,recall400 = 0.0,f1Score400 = 0.0,mcc = 0.0)
+                override fun estimateOverallMetrics(): ModelEvaluation  = ModelEvaluation(accuracy = 0.5,precision400 = 0.5,recall400 = 0.0,f1Score400 = 0.0,mcc = 0.0)
```

Comment:
duplicated code. this default instance should really be instantiated only once and referred in all thes different places. so that, if we want to change the default of one parameter, we do it only once, and not 20 times :)